### PR TITLE
[ ci ] install mingw64 gcc for windows

### PR DESCRIPTION
**Pull request by joelberkeley**
~~currently commented out to check CI is actually broken~~ not any more

### CODE REVIEW

**Comment by joelberkeley** [comment]
**I DON'T THINK CI RAN, SO MAYBE DON'T MERGE YET?**

File: .github/workflows/ci-idris2.yml [hunk 1]
```diff
@@ -201,7 +201,7 @@ jobs:
       - name: Get Chez Scheme
         run: |
           git clone --depth 1 https://github.com/cisco/ChezScheme
-          c:\msys64\usr\bin\bash -l -c "pacman -S --noconfirm tar make"
+          c:\msys64\usr\bin\bash -l -c "pacman -S --noconfirm tar make mingw-w64-x86_64-gmp"
```
Comment (benhormann) [inline .github/workflows/ci-idris2.yml:204]:
```suggestion
          c:\msys64\usr\bin\bash -l -c "pacman -S --noconfirm tar make mingw-w64-x86_64-gcc"
```

GCC depends on GMP. So if `mingw-w64-x86_64-gmp` is missing, we need to install `mingw-w64-x86_64-gcc`.
Add it for windows-self-host-racket too, line 419.

BTW, the first warning gives a clue to this: `idris_net.c:211:3: warning: implicit declaration of function 'inet_ntop'; did you mean 'inet_ntoa'?` (normally there are no warnings).

**Review by benhormann — CHANGES_REQUESTED** [review]
Seems the MINGW64 compiler is no longer installed by default?

File: .github/workflows/ci-idris2.yml (hunk 1 above)
Comment (gallais) [reply .github/workflows/ci-idris2.yml]: Let's try that!

Comment (benhormann) [reply .github/workflows/ci-idris2.yml]: @gallais The other windows job will need it too. (pity I can't add a suggestion there too)

**Comment by benhormann** [comment]
```
git reset 1011cc61 --soft
git commit -m 'install mingw64 gcc for windows CI jobs'
git push -f
```
would clean up these commits.

Though gallais will probably squash the PR merge, so maybe just update the title instead?

**Comment by joelberkeley** [comment]
@benhormann i believe the person merging can choose the commit message

**Comment by gallais** [comment]
Thank you both for looking into this!

**Pull request merged**